meta-tx/test/MetaTransaction.test.js

```diff
@@ -122,6 +148,30 @@ contract("ERC721MetaTxTokenTestAllien", accounts => {
     assert.isTrue(newNonce.toNumber() == nonce + 1, "Nonce not incremented");
   });
 
+  it("Call known abi-method(getNonce), use metaTx, for check contract support metaTx", async () => {
+  	let nonce = await metaTxTest.getNonce(publicKey);
+  	let {
+      r,
+      s,
+      v,
+      functionSignature
+    } = await getTransactionData(nonce, getNonceAbi, [ZERO_ADDRESS]);
+    let resultExecMataTx = await metaTxTest.executeMetaTransaction(publicKey, functionSignature, r, s, v, {from: accounts[0]})
+
+    var newNonce = await metaTxTest.getNonce(publicKey);
+    assert.isTrue(newNonce.toNumber() == nonce + 1, "Nonce not incremented");
+  });
+
+  it("Check contract supports metaTransaction by method areMetaTxSupported, use contract yes MetaTx", async () => {
+  	let result = await areMetaTxSupported(metaTxTest);
```

Comment:
it would be cool to check if nonce is the same after areMetaTxSupported() executed

Reply:
also we could check areMetaTxSupported() for contract that doesn't have getNonce()

Reply:
ok, done